.github/workflows/layer_govcloud.yml

```diff
@@ -0,0 +1,141 @@
+on:
+  workflow_dispatch:
+    inputs:
+      environment:
+        description: Deployment environment
+        type: choice
+        options:
+          - Gamma
+          - Prod
```

Comment:
This pipeline should be executed in both environments simultaneously, otherwise we may deploy in one environment and forget the other. Is it possible to add a "both" option and make it the default or do we remove it?

Reply:
This workflow will be converted to a dispatch workflow so we can use it in the main release workflow eventually. Leaving it as per env is better for that purpose.

Reply:
A `workflow_call` option will be added as well as this so when we're happy with the performance, we can add it to the release workflow. I'm going to leave this as per env for that reason.